about.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="En">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="author" content="Sergio Valadez">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="css/main.css">
+  <link rel="stylesheet" href="css/nav.css">
+  <link rel="stylesheet" href="css/style.css">
+
+  <title>Agro-industry 2021</title>
+  <link rel="icon" href="/images/logo3.png">
+</head>
+
+<body>
+  <header class="topnav">
+    <a><img src="./images/ic_github.png" alt="facebook logo"></a>
+    <a><img src="./images/ic_twitter.png" alt="twitters logo"></a>
+    <a href="#">Spanish</a>
+    <a href="#">My Page</a>
+    <a href="#">Log Out</a>
+  </header>
+  <header>
+    <nav>
+      <a href="index.html"><img class="logo" src="./images/logo.svg" alt="logo"></a>
```

Comment:
- [ ] I can see that you have the correct path as requested by the previous reviewer Thanks for making that change 👏. I notice it's still not working on your deployed version but it is working when I clone your project. I suspect this is due to the fact you are deploying Github pages from the wrong branch. so please check this and confirm that you are deploying from the branch with the current changes you've made 👍

Reply:
That is accurate. The branch I'm merging is named "milestone1". It also is the branch that is selected to be displayed on the live demo. Please, double check on that. In my end everything is running smoothly.

Reply:
Github pages just deployed right now. I think your previous deploys had a problem because the didn't fully deploy. this is resolved now